controller/users_blackbox_test.go

```diff
@@ -72,7 +72,9 @@ func (s *TestUsersSuite) SecuredServiceAccountController(identity account.Identi
 func (s *TestUsersSuite) TestUpdateUserOK() {
 	// given
 	user := s.createRandomUser("TestUpdateUserOK")
-	identity := s.createRandomIdentity(user, account.KeycloakIDP)
+	identity, err := testsupport.CreateTestUser(s.DB, &user)
```

Comment:
Isn't this creating user twice?

Reply:
`createRandomUser` only creates the user object ( but not in db ).
`CreateTestUser` creates the user in the db. 

( we should refactor the method name maybe )

Reply:
Cool , i am gonna rename this to `giveUser`. There are other files which has the same method, but that saves user in the database as well